packages/retail-ui/components/Sticky/Sticky.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import LayoutEvents from '../../lib/LayoutEvents';
-import { createPropsGetter } from '../internal/createPropsGetter';
 import { Nullable } from '../../typings/utility-types';
 import styles from './Sticky.module.less';
 import { isFunction } from '../../lib/utils';
 import { cx } from '../../lib/theming/Emotion';
 import warning from 'warning';
+import shallowEqual from 'fbjs/lib/shallowEqual';
 
 export interface StickyProps {
   side: 'top' | 'bottom';
-  offset?: number;
+  offset: number;
```

Comment:
Если делать offset  обязательным, может стоит ему значение по умолчанию сделать?

Reply:
Ты имеешь в виду в доку добавить? Да, согласен
```suggestion
  /**
   * Отступ в пикселях от края экрана, на сколько сдвигается элемент в залипшем состоянии
   * @default 0
   */
  offset: number;
```